docs/site/src/styles/global.css

```
@@ -0,0 +1,430 @@
+  --color-border: #30363d;
+  --color-border-bright: #424b56;
+  --color-text: #e6edf3;
+  --color-muted: #8b949e;
+  --color-accent: #58a6ff;
+  --color-accent-2: #a371f7;
+  --color-code-bg: #1f242c;
+  --color-good: #3fb950;
+  --color-bad: #f85149;
+  --color-baseline: #d29922;
+  --font-sans:
+    ui-sans-serif, system-ui, -apple-system, "Segoe UI", Roboto, "Helvetica Neue", Arial,
```

Stylelint (17.11.1)

[error] Expected "Roboto" to be "roboto" (value-keyword-case)

(value-keyword-case)